cirq/contrib/routing/utils.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Callable, Iterable, List, Optional
+
+import networkx as nx
+
+from cirq import circuits, ops
+import cirq.contrib.acquaintance as cca
+from cirq.contrib.routing.swap_network import SwapNetwork
+
+BINARY_OP_PREDICATE = Callable[[ops.Operation, ops.Operation], bool]
+
+
+def get_timeslices(dag: circuits.CircuitDag) -> List[nx.Graph]:
```

Comment:
docstring. consider adding something about the fact that it's giving you graphs somewhere in the function name

Reply:
Done